src/hotspot/share/runtime/objectMonitor.cpp

```diff
@@ -1397,37 +1357,32 @@ void ObjectMonitor::ExitEpilog(Thread * Self, ObjectWaiter * Wakee) {
 // The _owner field is not always the Thread addr even with an
 // inflated monitor, e.g. the monitor can be inflated by a non-owning
 // thread due to contention.
-intx ObjectMonitor::complete_exit(TRAPS) {
-  Thread * const Self = THREAD;
-  JavaThread * jt = Self->as_Java_thread();
```

Comment:
Wow! 3 different ways to refer to the same thread in one function: THREAD, Self and jt.
Thanks for cleaning that up.